src/screens/profile/CreateAmenities/CreateAmenities.utils.ts

```diff
@@ -0,0 +1,154 @@
+import { IconName, IconSet } from 'src/types';
+
+export type AmenitiesErrorHandler = {
+  error: boolean;
+  message: string;
+};
+
+export const ERROR_NONE: AmenitiesErrorHandler = {
+  error: false,
+  message: '',
+} as const;
+
+export const amenityFormValidation = (inputValue: string): AmenitiesErrorHandler => {
+  if (inputValue.length < 2) {
```

Comment:
2 and 30 are magic numbers, put them into constants variables

Reply:
added contants for min/max values